src/components/biggive-accordion/biggive-accordion.tsx

```diff
@@ -0,0 +1,59 @@
+import { Component, Prop, Element, h } from '@stencil/core';
+import { brandColour } from '../../globals/brand-colour';
+import { spacingOption } from '../../globals/spacing-option';
+
+@Component({
+  tag: 'biggive-accordion',
+  styleUrl: 'biggive-accordion.scss',
+  shadow: true,
+})
+export class BiggiveAccordion {
+  @Element() host: HTMLBiggiveAccordionElement;
+
+  @Prop() spaceBelow: spacingOption = 0;
+
+  @Prop() textColour: brandColour = 'black';
+
+  @Prop() headingColour: brandColour = 'primary';
+
+  children: Array<HTMLBiggiveAccordionEntryElement> = [];
+
+  componentWillLoad() {
+    this.children = Array.from(this.host.children) as Array<HTMLBiggiveAccordionEntryElement>;
+  }
+
+  expandHandler(e) {
+    let entry = e.target.closest('.entry');
+    let arrow = entry.querySelector('.arrow');
+
```

Comment:
```suggestion
    const entry = e.target.closest('.entry');
    const arrow = entry.querySelector('.arrow');

```

I think it makes sense to always use const instead of let where possible - let me know if you disagree.

Reply:
Updated as per the suggestion